### PR TITLE
Make tests pass with current versions of things

### PR DESCRIPTION
**Pull request by shanemcd**

### CODE REVIEW

**Review by rooftopcellist — APPROVED** [review]
lgtm

File: awx/conf/tests/unit/test_registry.py [hunk 1]
```diff
@@ -29,9 +29,10 @@ def reg(request):
     # as "defined in a settings file".  This is analogous to manually
     # specifying a setting on the filesystem (e.g., in a local_settings.py in
     # development, or in /etc/tower/conf.d/<something>.py)
-    defaults = request.node.get_marker('defined_in_file')
-    if defaults:
-        settings.configure(**defaults.kwargs)
+    for marker in request.node.own_markers:
+        if marker.name == 'defined_in_file':
+            settings.configure(**marker.kwargs)
```
Comment (ryanpetrello) [inline awx/conf/tests/unit/test_registry.py:34]: Well this is annoying.  What a dumb change to this API.

File: awx/main/tests/docs/test_swagger_generation.py [hunk 2]
```diff
@@ -79,10 +77,14 @@ def _prepare(self, get, admin):
             data['paths'] = revised_paths
             self.__class__.JSON = data
 
-    def test_sanity(self, release):
+    def test_sanity(self, release, request):
         JSON = self.__class__.JSON
         JSON['info']['version'] = release
 
+
+        if not request.config.getoption('--genschema'):
```
Comment (ryanpetrello) [inline awx/main/tests/docs/test_swagger_generation.py:85]: 👍

**Review by ryanpetrello — APPROVED** [review]

**Comment by ryanpetrello** [comment]
regate

**Pull request merged**